packages/components/src/Link/Link.tsx

```diff
@@ -20,6 +21,7 @@ export function Link({
       {...(external && { target: "_blank" })}
       {...(ariaLabel && { "aria-label": ariaLabel })}
       {...(ariaExpanded && { "aria-expanded": ariaExpanded })}
+      className={styles.link}
```

Comment:
I went with the css style route vs using the typography component so I could add the underline. But if you think I should go with the typography component I can switch it up!

Reply:
this works too!